aslprep/cli/parser.py

```diff
@@ -202,9 +202,11 @@ def _bids_filter(value, parser):
         type=str,
         nargs='+',
         help=(
-            'Search PATH(s) for pre-computed derivatives. '
+            'Search PATH(s) for pre-computed BIDS derivatives. '
             'These may be provided as named folders '
-            '(e.g., `--derivatives smriprep=/path/to/smriprep`).'
+            '(e.g., `--derivatives smriprep=/path/to/smriprep`). '
+            'Note: If your FreeSurfer outputs are not BIDS-compliant '
+            '(e.g., not created by sMRIPrep), use `--fs-subjects-dir` instead.'
```

Comment:
This might be a little misleading- I believe there was an attempt to convert FS outputs into a BIDS derivatives dataset at some point. sMRIPrep doesn't really do that though- it just grabs the parts it thinks are important and leaves the FS directory as-is.
```suggestion
            'Note: This does not apply to non-BIDS data, like FreeSurfer outputs. '
            'To re-use FreeSurfer outputs, use `--fs-subjects-dir` instead.'
```